scripts/package_building/build_artifacts.sh

```diff
@@ -441,7 +443,21 @@ function build_debian (){
     if [[ "$build_state" == "kernel" ]];then
         pushd "$source"
         make-kpkg --rootcmd fakeroot --initrd  --revision "$build_date" -j"$thread_number" kernel_image kernel_headers kernel_source kernel_debug
+	last_commit=$(git log --format="%H" -n 1)
```

Comment:
create a method --- build_kernel_metapackage_deb and put all this code in there, parameterized with the required things